src/main/kotlin/no/nav/familie/tilbake/api/dto/FagsakDto.kt

```diff
@@ -1,17 +1,28 @@
 package no.nav.familie.tilbake.api.dto
 
+import no.nav.familie.kontrakter.felles.PersonIdent
+import no.nav.familie.tilbake.behandling.domain.Behandlingsstatus
+import no.nav.familie.tilbake.behandling.domain.Behandlingstype
 import no.nav.familie.tilbake.behandling.domain.Fagsaksstatus
 import no.nav.familie.tilbake.behandling.domain.Ytelsestype
 import no.nav.familie.tilbake.integration.pdl.internal.Kjønn
 import java.time.LocalDate
+import java.util.UUID
 
 data class FagsakDto(val eksternFagsakId: String,
                      val status: Fagsaksstatus,
                      val ytelsestype: Ytelsestype,
-                     val søkerFødselsnummer: String,
+                     val fagsystemskode: String,
```

Comment:
Ikkje kall det fagsystemskode i dtoen. For frontend er fagsystem berre BA/EF/KS, så den bør heite fagsystem!

Reply:
endret til fagsystem